bin/gftools-add-font.py

```diff
@@ -259,10 +261,28 @@ def main(argv):
   if os.path.isfile(desc):
     print('DESCRIPTION.en_us.html exists')
   else:
-    _WriteTextFile(desc, 'N/A')
+    html = '''<p>$DESCRIPTION</p>
+
+<p>To contribute, see <a href="$UPSTREAMREPO">$UPSTREAMREPO_WITHOUT_HTTP</a></p>.
```

Comment:
"." shoud be within `</p>`

Reply:
Yup. It was meant outside the <a>. Corrected it.